.gitmodules

```diff
@@ -1,4 +0,0 @@
-[submodule "dev-lib"]
-	path = dev-lib
-	url = https://github.com/xwp/wp-dev-lib.git
-	branch = master
```

Comment:
This file needs to be restored.

Reply:
I deleted it because it was blocking my ability to commit and I thought my pull request would not have included the deletion. I am not exactly sure what I was doing wrong or how to fix it _(I gave up on submodules long ago as I always had issues with them.)_ 

Is there any chance that you can accept the pull request after I fix everything else, and then restore manually?

Reply:
The lack of the submodule is causing Travis CI to fail: https://travis-ci.org/Automattic/amp-wp/jobs/313361694#L186

So it needs to be restored before we merge the PR so that the automated tests and checks can be performed.

What is the error you're having?

Reply:
Let me see if I can figure it out.  

I was using SourceTree to allow me to make partial commits of files and it said I needed the submodule but was locked in the dialog and would not let me exit.  When I deleted the file SourceTree recognized the change and allowed my commit to continue.

I will try reverting this commit and see if I can get things to work from the command line.  If I have an issue I will let you know otherwise I will resolve it and we can just move forward.